src/utils/dashboardUtils.ts

```diff
@@ -164,12 +164,13 @@ export const createQueryParams = (filters: any) => {
   return queryParams.toString();
 };
 
-export const getRestorationGoalResumeData = (data: DashboardTreeRestorationGoalResponse) => {
-  return [
+export const getRestorationGoalResumeData = (data: TreeRestorationGoalDto) => {
+  const result = [
     { name: "Total", value: data?.totalTreesGrownGoal, color: "#13487A" },
     { name: "Enterprise", value: data?.forProfitTreeCount, color: "#7BBD31" },
     { name: "Non Profit", value: data?.nonProfitTreeCount, color: "#B9EDFF" }
   ];
+  return result;
```

Comment:
No reason to assign to a variable just to turn around and return it (my IDE would even give a soft warning on this line because of it). I'd either go back to the `return [` statement, or streamline this even more and remove the return:
```
export const getRestorationGoalResumeData = (data: TreeRestorationGoalDto) => [
  ...
];
```